src/framebuffer.ml

```diff
@@ -445,6 +452,7 @@ let draw_char (x : int) (y : int) (f : Font.t) (c : char) (col : int) (buffer :
   )
 
 let draw_string (x : int) (y : int) (f : Font.t) (s : string) (col : int) (buffer : t) =
+  buffer.dirty <- true;
```

Comment:
This doesn't modify the buffer, but calls something that does, so let that do the work.